stetho/src/main/java/com/facebook/stetho/inspector/network/AsyncPrettyPrinter.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2014-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+package com.facebook.stetho.inspector.network;
+
+public interface AsyncPrettyPrinter {
+  String print(byte[] payload);
```

Comment:
Strongly prefer a more efficient API:

  void printTo(PrintWriter output, InputStream payload) throws IOException;

You can internally use a byte[] and String but this API lets you optimize later.